scoringrules/visual/reliability.py

```diff
@@ -0,0 +1,112 @@
+import typing as tp
+
+import numpy as np
+from scipy.optimize import isotonic_regression
+from scipy.interpolate import interp1d
+from scipy.stats import bernoulli
+
+
+def reliability_diagram(
+    observations: np.ndarray,
+    forecasts: np.ndarray,
+    /,
+    uncertainty_band: tp.Literal["confidence", "consistency"] | None = "consistency",
+    n_bootstrap: int = 100,
+    alpha: float = 0.05,
+):
+    """Plot the reliability diagram of a set of predictions.
+    
+    CORP: Consistent, Optimally binned, Reproducible, PAV-algorithm based
+    reliability diagram from 
+    [Dimitriadis et al. (2021)](https://www.pnas.org/doi/full/10.1073/pnas.2016191118)
+    
+    Parameters
+    ----------
+    observations: ArrayLike
+        The observed values.
+    forecasts: Array
+        The predicted forecast ensemble, where the ensemble dimension is by default
```

Comment:
The code suggests `forecasts` is a vector of probability forecasts rather than an ensemble?

Reply:
I must have copy pasted the docstring when I started working on it and forgot to change it lol